lib/generators/asset_sync/templates/asset_sync.rb

```diff
@@ -81,6 +81,9 @@
     # Path to cache file to skip scanning remote
     # config.remote_file_list_cache_file_path = './.asset_sync_remote_file_list_cache.json'
     #
+    # Path to maintain remote file list in remote
```

Comment:
I think we need better explanation here...
Something like `Path on remote storage to persist remote file list file`?

Already I am confused here too

Reply:
Updated the suggestion